Document XSLTProcessor.* (#25563)
mdn/content: files/en-us/web/api/xsltprocessor/transformtofragment/index.md

---
title: XSLTProcessor.transformToFragment()
slug: Web/API/XSLTProcessor/transformToFragment
page-type: web-api-instance-method
browser-compat: api.XSLTProcessor.transformToFragment 
---

{{APIRef("XSLT")}}

> The `transformToFragment()` method of the {{domxref("XSLTProcessor")}} interface transforms the {{DOMxRef("Node")}} source given in parameter using the XSLT stylesheet associated with `XSLTProcessor` with the {{domxref("XSLTProcessor.importStylesheet")}}.

Reviewer: I think?

```suggestion
The `transformToFragment()` method of the {{domxref("XSLTProcessor")}} interface transforms a provided {{DOMxRef("Node")}} source to a {{domxref("DocumentFragment")}} using the XSLT stylesheet associated with the `XSLTProcessor`.
```
